src/input_buttons.h

```diff
@@ -94,6 +170,186 @@ namespace Input {
 				return false;
 		}
 	}
+
+	namespace Direction {
+		enum InputDirection : uint8_t {
+			NONE = 0,
+			DOWNLEFT = 1,
+			DOWN = 2,
+			DOWNRIGHT = 3,
+			LEFT = 4,
+			CENTER = 5,
+			RIGHT = 6,
+			UPLEFT = 7,
+			UP = 8,
+			UPRIGHT = 9,
+			NUM_DIRECTIONS = 10,
+		};
+
+		static constexpr auto kNames = makeEnumTags<InputDirection>(
+				"NONE",
+				"DOWNLEFT",
+				"DOWN",
+				"DOWNRIGHT",
+				"LEFT",
+				"CENTER",
+				"RIGHT",
+				"UPLEFT",
+				"UP",
+				"UPRIGHT",
+				"NUM_DIRECTIONS");
+	};
```

Comment:
indent looks off here

Reply:
Since it's function call parameters my vim wants to double indent it. Fixed manually.